.github/dependabot.yml

```diff
@@ -6,22 +6,23 @@ updates:
       interval: daily
       time: "11:00"
     open-pull-requests-limit: 10
-  - package-ecosystem: npm
-    directory: "/docs"
-    schedule:
-      interval: daily
-      time: "11:00"
-    open-pull-requests-limit: 10
-    reviewers:
-      - fadeev
+#  - package-ecosystem: npm
```

Comment:
do we really need to keep these commented lines or can we delete its?

Reply:
I assume we might want to re-enable this in future (when we publish our documentation)